.github/workflows/tag_module_cleanup.yml

```diff
@@ -19,17 +19,20 @@ jobs:
       - uses: actions/github-script@a3e7071a34d7e1f219a8a4de9a5e0a34d1ee1293
         with:
           script: |
-            // 2021 q3 release milestone.
-            // https://github.com/google/blockly/milestone/18
-            const milestoneNumber = 18;
-            // Note that pull requests are accessed through the issues API.
-            const issuesUpdateParams = { 
-              owner: context.repo.owner, 
-              repo: context.repo.repo, 
-              // Adds the milestone
-              milestone: milestoneNumber, 
+            // Note that pull requests are considered issues and "shared"
+            // actions for both features, like manipulating labels and
+            // milestones are provided within the issues API.
+            await github.issues.update({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: context.issue.number,
+              // 2021 q3 release milestone.
+              // https://github.com/google/blockly/milestone/18
+              milestone: 18
```

Comment:
I think this should still be `milestoneNumber`

Reply:
`milestoneNumber` is a local variable that doesn't exist in this version.

Reply:
Yeah, I was trying to say that I think we should keep the local variable `milestoneNumber` around. Probably not worth opening a new pr though so it is fine : )